i18n/locales/source/blockly-mooc/applab.json

```diff
@@ -433,6 +433,8 @@
   "stepIn": "Step in",
   "stepOver": "Step over",
   "stepOut": "Step out",
+  "submitYourProject": "Submit your project",
+  "submitYourProjectConfirm": "You cannot edit your project after submitting it, really submit?",
```

Comment:
why do these strings show up here? is this the result of running some i18n script?